.yamato/config/universal.metafile

```diff
@@ -11,6 +11,14 @@ test_platforms:
       Win: 2000
   - type: playmode
   - type: editmode
+  - type: playmode
```

Comment:
i think this testplatform can be removed? i think it got here accidentally from my example branch